src/actions/index.js

```diff
@@ -0,0 +1,4 @@
+export const changeUsername = (username) => ({
```

Comment:
I think we should create a types.js file to keep all the types as constants and use them both in actions and reducers.
(e.g. 

// types.js
export const CHANGE_USERNAME = "CHANGE_USERNAME";

)